addon/components/frost-chart-y-axis.js

```diff
@@ -118,15 +119,21 @@ export default Component.extend({
 
   init () {
     this._super(...arguments)
-    this.attrs.dispatch({
+    this.dispatch({
       type: 'REGISTER_AXIS'
     })
   },
 
-  didInsertElement () {
-    this._super(...arguments)
-    run.scheduleOnce('afterRender', this, this._dispatchRenderedAxis)
-  }
-
   // == Actions ===============================================================
+  actions: {
+    _dispatchTickRendered (action) {
```

Comment:
I personally like the conversion so it's very simple to distinguish what's internal state/methods and what's being exposed.